.travis.yml

```diff
@@ -11,6 +11,7 @@ matrix:
       dist: trusty
     - python: 3.3
       dist: trusty
-script: python yapi.py install test
+script:
+  - python yapi.py install test
```

Comment:
It's probably need a script per every command of yapi to have more coverage.

Reply:
I will add that.